packages/gatsby-link/index.d.ts

```diff
@@ -1,11 +1,19 @@
-import * as React from "react";
+import * as React from "react"
+import { NavigateFn, LinkProps } from "@types/reach__router"
```

Comment:
CMIIW, but shouldn't it be importee from `@reach/router` instead? IIRC it will resolve to `@types/reach__router` anyway

Reply:
Oops, yeah, I meant to use `@reach/router` but somehow missed it. I'll make a follow-up PR to fix this.